01 Minimap Fixes/gamedata/scripts/modxml_dart_minimap.script

```diff
@@ -174,7 +174,7 @@ function adjustMap(DXML, settings)
   if align.x == "CENTER" then
     x = DART.aspectRatio4x3 / 2 + offset.x / DART.baseHeight / DART.getAspectRatioVs4x3()
   elseif align.x == "RIGHT" then
-    x = DART.aspectRatio4x3 - (size.x / 2 - offset.x) / DART.baseHeight / DART.getAspectRatioVs4x3()
+    x = (DART.aspectRatio4x3 - (size.x / 2 - offset.x) / DART.baseHeight / DART.getAspectRatioVs4x3()) - (DART.HUD_SAFE_ZONE / 768)
```

Comment:
I think we can put the safe zone inside like so to simplify a bit:
```
x = DART.aspectRatio4x3 - (size.x / 2 - offset.x - DART.HUD_SAFE_ZONE) / DART.baseHeight / DART.getAspectRatioVs4x3()
```

(I'm not actually expecting you to revise this you've helped plenty already, just making notes).